src/Button.jsx

```diff
@@ -70,13 +70,14 @@ var Button = React.createClass({
   },
 
   renderNavItem: function (classes) {
+    classes['navbar-btn'] = true;
     var liClasses = {
       active: this.props.active
     };
 
     return (
       <li className={classSet(liClasses)}>
-        {this.renderAnchor(classes)}
+        {this.renderButton(classes)}
```

Comment:
Instead of changing this to blindly render a button we should extract the logic on when to use a button vs an anchor and do the right one. That logic is here: https://github.com/react-bootstrap/react-bootstrap/blob/v0.15.0/src/Button.jsx#L39-L40